scripts/annotations/PrefixColumns.java

```diff
@@ -0,0 +1,54 @@
+package annotations;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
+import static annotations.CSVUtils.*;
+
+/**
+ * Simply prefixes (or postfixes) the content of every cell 
+ * in specified column (index).
+ */
+public class PrefixColumns {
+    
+    public static void main(String[] args) throws Exception {
+        
+        // BEGIN Parameters
+        
+        String in = "idr0043-uhlen-humanproteinatlas/experimentA/idr0043-experimentA-filePaths.tsv";
+        String out = "idr0043-uhlen-humanproteinatlas/experimentA/idr0043-experimentA-filePaths_2.tsv";
+        
+        // prefix or postfix
+        boolean prefix = true;
+        
+        // the text to use as prefix/postfix
+        String concat = "Dataset:name:";
+        
+        // the column index
+        int index = 0;
+        
+        char separator = '\t';
+        
+        // END Parameters
+        
+        BufferedReader r = new BufferedReader(new FileReader(in));
```

Comment:
Maybe user try-with-resources

Reply:
👍 I have to do a general overhaul of the scripts, they are still pretty "quick and dirty". As there will be many HPA projects coming in (in hopefully exactly the same format),  ideal would be to finally have one script which generates all necessary files (filePaths.tsv, annotation.csv, etc.) in one go.